7-gce/gce/deploy.sh

```diff
@@ -20,7 +20,8 @@ ZONE=us-central1-f
 GROUP=frontend-group
 TEMPLATE=$GROUP-tmpl
 MACHINE_TYPE=f1-micro
-IMAGE=debian-8
+IMAGE_FAMILY=debian-8
+IMAGE_PROJECT=debian-cloud
```

Comment:
Param _-IMAGE_ is deprecated. Updated to use _-IMAGE_FAMILY_ and _-IMAGE_PROJECT_.